src/utils/read_from_file/read_from_file.cpp

```diff
@@ -1,5 +1,6 @@
 #include "read_from_file.h"
 #include "./handle_battle_cmd.h"
+#include "./handle_tutorial_battle_cmd.h"
```

Comment:
Please keep the imports in alphabetical order.
Also, it might be a good idea to align with the `./` in the front of the neighboring files; either all of them will have it, or none of them.